gradle/dependencies.gradle

```diff
@@ -183,7 +183,7 @@ ext {
       dependencyGraph      : '0.5.0',
       dependencyUpdates    : '0.29.0',
       kotlin               : kotlinVersion,
-      license              : '0.8.80',
+      license              : '0.8.90',
```

Comment:
The point is that all our builds suddenly started failing, because it can't resolve a transitive dependency "org.jetbrains.kotlinx:kotlinx-html-jvm:0.7.1", which is required by com.jaredsburrows:gradle-license-plugin:0.8.80".  It really doesn't exist in all the repos we specified (but other versions do exist). If you clear gradle caches, it also starts failing locally. Looks like the version was removed from some of the repos just yesterday or it used to work on caches for some time.
The [docs for kotlinx-html-jvm](https://github.com/kotlin/kotlinx.html/wiki/Getting-started) say we should use the repo that I added to our buildscript (and it does have more versions than [maven](https://repo.maven.apache.org/maven2/org/jetbrains/kotlinx/kotlinx-html-jvm/), so I think we should leave this new repo, maybe they'll remove more versions from maven in the future). Also, I had to bump the license dependency, because the minimum version in the official repo is 0.7.2, but 0.8.80 needed 0.7.1.

Reply:
CP to main: https://github.com/mapbox/mapbox-navigation-android/pull/7301